### DISCUSSION TITLE
How would you use WebSockets with Falco?

### DISCUSSION THREAD

**Discussion by johanalkstal** [falcoframework/Falco#75] [Q&A]
How would I add WebSocket endpoints and broadcasting to clients with Falco?

I'm a newbie with dotnet so if the answer is obvious, I apologize before hand. 🙂

**Comment by pimbrouwers** [comment]
This is a great question! I will admit I am not an expert on this either. But I will certainly find out and add a section to the docs for it.

**Comment by pimbrouwers** [comment]
Going to add a sample project showing this.

**Comment by oeisalphanumeric** [comment]
Is there any documentation somewhere? Did not find a sample project.
